src/layer/highway_shield.js

```diff
@@ -19,6 +19,20 @@ function routeConcurrency(num) {
           ["concat", "\n", ["get", "name"]],
           "",
         ],
+        [
+          "match",
+          ["get", "route_" + num],
```

Comment:
Combine these match expressions with the Kentucky parkway match expression above. The line that says `US:KY:Parkway=` can contain an array of strings instead of a single string.